ui/src/components/yaml/styles.module.less

```diff
@@ -359,261 +356,277 @@
         }
       }
 
-      .yaml_content_streamingIndicator {
-        margin: 18px 0;
-        display: flex;
-        align-items: center;
-        gap: 4px;
-
-        .dot {
-          width: 6px;
-          height: 6px;
-          background-color: #4447c3;
-          border-radius: 50%;
-          opacity: 0.3;
-          animation: dotPulse 1.4s infinite;
-
-          &:nth-child(2) {
-            animation-delay: 0.2s;
-          }
+      .yaml_content_diagnosisContent {
+        color: #d4d4d4;
+        background: #2b1d3c;
+        border-radius: 0 0 12px 12px;
+        padding: 16px;
+        font-size: 14px;
```

Comment:
The loading state's height is set to 100% but may not behave as expected when nested inside .yaml_content_diagnosisContent. Consider explicit sizing or parent constraints.